_guides/002-install_and_run.md

```diff
@@ -14,17 +14,34 @@ sections:
     module: install/prerequisites
     items:
       -
-        title: Cluster discovery
-        description: Cluster discovery
+        title: Cluster Discovery
+        description: Cluster Discovery
         module: install/prerequisites/clusterdiscovery
       -
-        title: Network Block Device
-        description: Network Block Device
-        module: install/prerequisites/nbd
+        title: Device Presentation
+        description: Device Presentation
+        module: install/prerequisites/devicepresentation
       -
         title: Ports and Firewall Settings
         description: Ports and Firewall Settings
         module: install/prerequisites/firewalls
+  -
+    title: Docker
+    description: Docker
+    module: install/docker
+    items:
+      -
+        title: Docker Container
+        description: Docker container install for Docker Engine 1.10+
+        module: install/docker/container
+      -
+        title: Docker Volume plugin
+        description: Docker managed plugin install for Docker Engine 1.13+
+        module: install/docker/plugin
+      -
+        title: Docker Swarm
+        description: Installing on Docker Swarm
+        module: install/schedulers/dockerswarm
   -
     title: Schedulers
```

Comment:
We should bring the Kubernetes and Docker Swarm docs up a level, then remove this Schedulers section entirely.